<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>977</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="day_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Day 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_day">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play One Day</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="message_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="entity_info_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="money_display">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cash: $0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_n_days">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play 1 Day</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="day_amount_entry">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdi">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>571</width>
      <height>881</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>-20</y>
      <width>20</width>
      <height>941</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionBreeding"/>
    <addaction name="actionTrading"/>
    <addaction name="actionPedigree"/>
    <addaction name="actionBuildings"/>
    <addaction name="actionEmployees"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionHorse_Properties"/>
   </widget>
   <widget class="QMenu" name="menuStartup">
    <property name="title">
     <string>Startup</string>
    </property>
    <addaction name="actionQuick"/>
    <addaction name="action10_Year"/>
    <addaction name="action20_Year"/>
    <addaction name="actionLoad_Saved"/>
   </widget>
   <addaction name="menuWindows"/>
   <addaction name="menuStartup"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBreed_Horses">
   <property name="text">
    <string>Breeding</string>
   </property>
  </action>
  <action name="actionBreeding">
   <property name="text">
    <string>Breeding</string>
   </property>
  </action>
  <action name="actionTrading">
   <property name="text">
    <string>Trading</string>
   </property>
  </action>
  <action name="actionPedigree">
   <property name="text">
    <string>Pedigree</string>
   </property>
  </action>
  <action name="actionBuildings">
   <property name="text">
    <string>Buildings</string>
   </property>
  </action>
  <action name="actionHorse_Properties">
   <property name="text">
    <string>Horse properties</string>
   </property>
  </action>
  <action name="actionEmployees">
   <property name="text">
    <string>Employees</string>
   </property>
  </action>
  <action name="actionQuick">
   <property name="text">
    <string>Quick</string>
   </property>
  </action>
  <action name="action10_Year">
   <property name="text">
    <string>10 Year</string>
   </property>
  </action>
  <action name="action20_Year">
   <property name="text">
    <string>20 Year</string>
   </property>
  </action>
  <action name="actionLoad_Saved">
   <property name="text">
    <string>Load Saved</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
